src/lib/ThumbnailsSidebar.js

```diff
@@ -51,16 +92,29 @@ class ThumbnailsSidebar {
 
         this.thumbnailImageCache = null;
         this.pdfViewer = null;
+        this.currentThumbnails = [];
+        this.currentPage = null;
+
+        this.anchorEl.removeEventListener('click', this.thumbnailClickHandler);
     }
 
     /**
      * Initializes the Thumbnails Sidebar
      *
+     * @param {Object} options - options for the Thumbnails Sidebar
```

Comment:
If options is optional, as seen by the guard below, you can wrap the comment docs in square braces.

`@param {Object} [options] - ...`